learn/polkadot/polkadot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tanssi and Polkadot
+description: Introduction to polkadot
+---
+
+# Tanssi and Polkadot
+
+Tanssi is a protocol built whithin the polkadot ecosystem, but what is Polkadot and why is it the right choice ?
+
+!!! note "Polkadot in a nutshell"
+    Polkadot is an innovative chain of chains, designed to address some major challenges present in other existing networks providing a more scalable, interoperable, secure, and customizable ecosystem in the blockchain space.
+
+## Main Features of Polkadot
+
+Polkadot is layer zero blockchain, which means that it offers essential services to other layer one chains within the ecosystem, including security, consensus and interoperability/communication, leaving the implementation of specific use cases to the parachains while tackling these common problems:
+
+- **Scalability** - Polkadot's sharded model allows for the parallel processing of transactions across multiple parachains, significantly improving scalability and overall network throughput while lowering processing times and fees. 
+- **Interoperability** - While most blockchain networks operate in isolation, making it difficult for them to communicate and share assets, Polkadot provide native interoperability through XCM (Cross Consensus Message Format), connecting multiple chains facilitating asset transfer capabilities and information exchange.
+- **Governance and Upgrades** - Polkadot's OpenGov introduces a quite unique on-chain governance model, empowering token holders to participate in decision-making, where there are no first class citizens bearing more privileges, and allowing for efficient upgrades and protocol improvements.
+- **Security** - In smaller blockchain networks, security can be a concern due to lower levels of mining or staking participation. Polkadot’s model of shared security guarantees the same security level even for the smaller parachain within the ecosystem.
+- **Customizability/flexibility** - Polkadot's architecture encourages the creation of sovereign blockchains with high specialization, enabling developers to design chains with specific features and functionalities optimized for the unique requeriments of the use case.
```

Comment:
List items don't typically start with a capital letter unless it's a name like "Polkadot", otherwise "While" should be changed to "while" etc.

Also no punctuation at the end of list items